util.js

```diff
@@ -2,14 +2,20 @@
 
 exports.stringToBytes = string => [...string].map(character => character.charCodeAt(0));
 
+function readTarSum(buffer) {
+	return parseInt(buffer.toString('utf8', 148, 154).replace(/\0.*$/, '').trim(), 8); // Read sum in header
+}
+
+exports.readTarSum = readTarSum;
```

Comment:
Does this need to be a separate util?

Can `tarHeaderCheckumMatches` simply fail early if `!isNaN(readTarSum(buffer)`?

Reply:
Good point.